JS/cookie.js

```diff
@@ -1,6 +1,4 @@
-document.querySelector('#formSize').addEventListener('submit', function(e) {
-    e.preventDefault()
-    var value = document.querySelector('input[name="size"]:checked').value;
-    document.cookie = value;
-    window.location.href = 'canvas.html'
+document.querySelector('#formSize').addEventListener('submit', function() {
+    let value = document.querySelector('input[name="size"]:checked').value;
+    document.cookie = "size="+value+"; expires=Thu, 18 Dec 2050 12:00:00 UTC";
```

Comment:
what happens after the 18th Dec 2050? the app will stop working!

Reply:
have updated